### ISSUE TITLE
Performance affected by Default.World.Unity.Rendering.RenderMeshSystemV2

### ISSUE THREAD

**Issue by dansyrotyn** [reeseschultz/ReeseUnityDemos#15]
Hello!
I have just launched Your Performance test scene for 4000 agents, and as I can see, Performance is heavily affected by Default.World.Unity.Rendering.RenderMeshSystemV2 on my machine
If launch demo without any renderers, this problem does not occur, obviously
I also have a few demos with 4k units but I do not face such low performance issues and have no idea what is wrong with Your scene
Can You please help figure out?

**Comment by reeseschultz** [comment]
The project uses the [Hybrid Renderer](https://docs.unity3d.com/Packages/com.unity.rendering.hybrid@0.1/manual/index.html), which has recently taken [a performance hit](https://forum.unity.com/threads/ecs-rendermeshsystemv2-performance-seems-to-be-much-slower-in-unity-2019-3.819942/). According to the linked post, supposedly the performance should be much better in `2020.1`, which I haven't upgraded to yet. Last time I tried, it was too unstable.

I'm surprised the performance is so bad for you. While the `RenderMeshSystemV2` does not perform nearly as well as any of the other built-in systems, I just tested the performance demo on a laptop lacking a dedicated GPU, and it ran the demo at the maximum framerate in the Unity editor (60FPS) for 4k units. Seems like it can handle many more, no problem. So is the demo noticeably lagging for you or something? Or are you just concerned about the mentioned system in particular?

Also, what are the specs of the device you're using? I'd also like to know how the performance is if you build the project and run it independently of the editor.
